test/conformance/platform/urPlatformCreateWithNativeHandle.cpp

```diff
@@ -30,6 +30,46 @@ TEST_F(urPlatformCreateWithNativeHandleTest, Success) {
     }
 }
 
+TEST_F(urPlatformCreateWithNativeHandleTest, SuccessWithOwnedNativeHandle) {
+    for (auto platform : platforms) {
+        ur_native_handle_t native_handle = nullptr;
+        if (urPlatformGetNativeHandle(platform, &native_handle)) {
+            continue;
+        };
+
+        // We cannot assume anything about a native_handle, not even if it's
+        // `nullptr` since this could be a valid representation within a backend.
+        // We can however convert the native_handle back into a unified-runtime
+        // handle and perform some query on it to verify that it works.
+        ur_platform_native_properties_t props = {
+            UR_STRUCTURE_TYPE_PLATFORM_NATIVE_PROPERTIES, nullptr, true};
+        ur_platform_handle_t plat = nullptr;
+        ASSERT_SUCCESS(
+            urPlatformCreateWithNativeHandle(native_handle, &props, &plat));
+        ASSERT_NE(plat, nullptr);
```

Comment:
could also add a query like `UR_PLATFORM_INFO_NAME` and compare the result to the original platform

Reply:
Done